openedx/features/edly/utils.py

```diff
@@ -0,0 +1,18 @@
+from openedx.features.edly.models import EdlyUserProfile
+
+
+def user_has_edly_organization_access(request):
```

Comment:
Please add  tests for this method:

1. user have access to a valid site URL which is linked to that user
2. user has no access to a valid site URL but that site in not linked to the user

Reply:
Done